Services/Init/tpl.login.html

```diff
@@ -1,29 +0,0 @@
-
-<!-- BEGIN login_information -->
-<div class="small">{TXT_LOGIN_INFORMATION}</div>
-<!-- END login_information -->
-
-<div class="ilStartupSection">
-    <script type="text/javascript" src="Customizing/global/skin/unibe52/js/shibboleth.js"></script>
-    <script type="text/javascript" src="https://wayf.switch.ch/SWITCHaai/WAYF/embedded-wayf.js"></script>
-    
-    <!-- BEGIN shib_login_form -->
-    <!-- END shib_login_form -->
-    <!-- BEGIN cas_login -->
-    <p>
-        {CAS_LOGIN_FORM}
-    </p>
-    <!-- END cas_login_form -->
-    <!-- BEGIN ilias_login -->
-    <div class="taxLoginLocal" style="display: none;">
-        {LOGIN_FORM}
-    </div><!-- END ilias_login -->
-
-    <!-- BEGIN agreement -->
-    <p class="taxUserAgreement">
-        {USER_AGREEMENT}
-    </p>
-    <!-- END agreement -->
-</div>
-
-
```

Comment:
Why have you deleted the tpl.login.html in this place and added a new version in the imucbsubstyle folder? The login window shouldn't be changed through substyles.